src/reference/informative/informC04_component_reference.rst

```diff
@@ -8,4 +8,105 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+
+      Understanding component references
+
+    As with references to units and variables, the term "component reference" refers to both sides of the naming and calling transaction.
```

Comment:
What's a "naming and calling transaction" ???

Reply:
Changed all wording in dc25db5